index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const integerRegex = /^-?\d+$/;
+const srcsetRegex = /\s*([^,]\S*[^,](?:\s+[^,]+)?)\s*(?:,|$)/;
```

Comment:
Why such a complicated regex? If it really has to be like this, I would like to see a code comment that explains it.

Reply:
I renamed the regex to `imageCandidateRegex` to indicate that it matches loosely to an ["image candidate string"](https://html.spec.whatwg.org/multipage/images.html#image-candidate-string). It's the simplest way I can think of for now to solve the issue where there's no whitespace between two image candidate strings. I've included some comments above. Let me know if I can make it clearer!